code/modules/projectiles/ammunition/caseless/_caseless.dm

```diff
@@ -1,5 +1,5 @@
 /obj/item/ammo_casing/caseless
-	desc = "A caseless bullet casing."
+	desc = "caseless round"
```

Comment:
This is the description, not the name.

Reply:
Yeah, oops. I fixed it but then force-pushed over my changes.

Reply:
Looks like this was already fixed, actually. It's now `A caseless round.` Caseless casing makes no sense.